src/WhereConditions/Operator.php

```diff
@@ -20,6 +20,22 @@ public function enumDefinition(): string;
      */
     public function default(): string;
 
+    /**
+     * The default value if no has operator is specified.
+     *
+     * @example "GTE"
+     */
+    public function defaultHasOperator(): string;
+
+    /**
+     * The value of the default has operator.
+     *
+     * TODO: Find or create an easy way to get the real enum value from the enum options
```

Comment:
This can be solved by looking at the parsed result of the entire definition (or maybe the executable enum type). In principle:

```php
\GraphQL\Type\Definition\EnumType#getValue($defaultHasOperator)
```